packages/example-app-base/src/components/Page/Page.types.ts

```diff
@@ -66,6 +66,9 @@ export interface IPageProps<TPlatforms extends string = string> {
   /** (7) Properties table(s) as Markdown string. Mutually exclusive with `jsonDocs`. */
   propertiesTablesSources?: string[];
 
+  /** (8) The accessibility guidelines as Markdown string. */
```

Comment:
Please update the numbering of the subsequent sections :)

Reply:
Oh wait, actually based on the JSX I think this should be 7 and jsonDocs/propertiesTableSources should be 8 (but still renumber the subsequent ones)

Reply:
tbh, i'm not sure why these are numbered. happy to change the value to whatever you suggest :D

Reply:
I numbered them to try and give an idea of which props correspond to sections on the page, and what order they go in.